Framework/DataHandling/test/LoadEmptyInstrumentTest.h

```diff
@@ -1043,6 +1043,33 @@ class LoadEmptyInstrumentTest : public CxxTest::TestSuite {
     TS_ASSERT_EQUALS(1, detectorInfo.detectorIDs()[1])
   }
 
+  void test_compare_wish_idf_vs_nexus() {
+    // Now rerun
+    LoadEmptyInstrument alg;
+    alg.setChild(true);
+    alg.initialize();
+    alg.setPropertyValue("Filename", "WISH_Definition_10Panels.hdf5");
+    alg.setPropertyValue("OutputWorkspace", "dummy");
+    alg.execute();
+    Mantid::API::MatrixWorkspace_sptr wish_nexus =
+        alg.getProperty("OutputWorkspace");
+
+    // Now re-run
+    alg.setPropertyValue("Filename", "WISH_Definition_10Panels.xml");
+    alg.execute();
```

Comment:
Are you sure that the second load will not simply return the instrument from the `InstrumentDataService`, based on a matching name?